docs/development.md

```diff
@@ -18,46 +18,20 @@
 
 # Development
 
-## How to setup a dev environment
+## Build requirements
 
-Step 1: setup mysql. Install docker with docker-compose and run the below
-command when setting up first time:
+* Mac OS X or Linux
+* Java 17+, 64-bit
```

Comment:
17+ is only true after https://github.com/trinodb/trino-gateway/pull/156 is merged.